cgmes/cgmes-conversion/src/test/java/com/powsybl/cgmes/conversion/test/export/CgmesExportTest.java

```diff
@@ -672,5 +672,52 @@ void testCanGeneratorControl() throws IOException {
         }
     }
 
+    @Test
+    void networkWithoutControlAreaInterchange() throws IOException {
+        Network network = DanglingLineNetworkFactory.create();
+        assertEquals(0, network.getAreaCount());
+
+        try (FileSystem fs = Jimfs.newFileSystem(Configuration.unix())) {
+            Path tmpDir = Files.createDirectory(fs.getPath("/temp"));
+            String baseName = "dl-net";
+
+            // Exporting with default behaviour, no default control area is written
+            Path tmpDirNoCA = tmpDir.resolve("network-no-ca");
+            Files.createDirectories(tmpDirNoCA);
+            network.write("CGMES", null, tmpDirNoCA.resolve(baseName));
+            Network networkNoCA = Network.read(new GenericReadOnlyDataSource(tmpDirNoCA, baseName));
+            assertEquals(0, networkNoCA.getAreaCount());
+
+            // Explicit creation of a default control area
+            new CgmesExport().createDefaultControlAreaInterchange(network);
+
+            // Check that exported files now have a control area definition
+            // No default value for tolerance
+            Path tmpDirWithCA = tmpDir.resolve("network-with-ca");
+            Files.createDirectories(tmpDirWithCA);
+            network.write("CGMES", null, tmpDirWithCA.resolve(baseName));
+            Network networkWithCA = Network.read(new GenericReadOnlyDataSource(tmpDirWithCA, baseName));
+            assertEquals(1, networkWithCA.getAreaCount());
+            Area areaExported = networkWithCA.getAreas().iterator().next();
+            assertEquals(1, areaExported.getAreaBoundaryStream().count());
+            assertEquals(-50, areaExported.getInterchangeTarget().orElse(Double.NaN));
+            // No default value for tolerance
+            assertNull(areaExported.getProperty(CgmesNames.P_TOLERANCE));
```

Comment:
Likewise, I recommend to not export and reimport, but rather read the exported files:

```suggestion
            eqFile = ConversionUtil.writeCgmesProfile(network, "EQ", tmpDir);
            String sshFile = ConversionUtil.writeCgmesProfile(network, "SSH", tmpDir);
            assertTrue(eqFile.contains("<cim:ControlArea rdf:ID=\"_dangling-line_N_CA\">"));
            assertTrue(sshFile.contains("<cim:ControlArea.netInterchange>-50</cim:ControlArea.netInterchange>"));
            // No default value for tolerance
            assertFalse(sshFile.contains("cim:ControlArea.pTolerance"));
```